test/Makefile

```diff
@@ -29,3 +29,8 @@ indent-cases:
 
 literate-cases:
 	${EMACS} -Q --script generate-literate-cases
+
+.PHONY: compile
+compile:
+	@cd ../lisp && make -k all \
```

Comment:
Let's add `compile` in lisp sub-folder for consistency.

Reply:
Done